pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -48,6 +48,7 @@ import (
 const (
 	randomMacGenerationAttempts = 10
 	allowForwarding             = 1
+	tapOwner                    = "0"
```

Comment:
I would call it `libvirtUserAndGid`.

Reply:
If it changes (GID & UID differ), we'd have to rename the variable. I don't think that's wise.